packages/xstate-wallet/src/channel-wallet2.ts

```diff
@@ -0,0 +1,202 @@
+import {
```

Comment:
How much of this code is copy/paste?

It is difficult to review the diff, without opening up two files side by side. Should I instead simply review the content of `channel-wallet2.ts`, ignoring the content of `browser-wallet.ts` that was deleted?

Reply:
I would recommend **not** reviewing `channel-wallet2.ts`. This file was removed in https://github.com/statechannels/statechannels/pull/3410/commits/d8a62b72ac6b0150917d967738e7667c369ecec4. The commit history could use a cleanup.

In this PR, the code added to `channel-wallet.ts` is a copy/paste from `browser-wallet.ts`.